### DISCUSSION TITLE
Serverless computing

### DISCUSSION THREAD

**Discussion by rajtilakjee** [vercel/community#4983] [Feature Question] [answered]
### Question

Does Vercel offer serverless computing like AWS Lambda? If yes, then my second question is, does it offer non-persistent storage in the same?

**Comment by swarnava** [comment, marked as answer]
Hi Rajtilak,

Vercel does offer Serverless Computing using [Serveless function](https://vercel.com/docs/functions/serverless-functions). For different storage options, you can review our Storage products: https://vercel.com/docs/storage
